README.rst

```diff
@@ -71,13 +71,26 @@ Allows changing configuration details of a given repository
 Installation
 ============
 
-From System Package Manager
----------------------------
+From System Repository 
+----------------------
 
 If your operating system packages repolib, you can install it by running::
-    
+
     sudo apt install python3-repolib
 
+From Pre-Compiled .deb Package
+------------------------------
+
+When using any other Debian based distro, it's possible to install the
+`latest release`__ using the `quick_install.sh`__ script:
+
+.. code:: shell
+
+    sudo apt update && sudo apt install --yes --no-install-recommends curl
+    sudo bash <(curl -s https://raw.githubusercontent.com/pop-os/repolib/HEAD/quick-install.sh)
```

Comment:
I don't want our README telling people to run a web script as root. Please update the wording for this section to what I suggested in https://github.com/pop-os/repolib/pull/70#discussion_r1294916406.